Portal/Services/WasteService.cs

```diff
@@ -126,15 +126,15 @@ public async Task SaveSelectedMonth(WasteTypesViewModel wasteTypesViewModel)
                 wasteTypesViewModel.SelectedWasteTypeId.Value);
         }
 
-        public async Task SaveSelectedWasteType(int journeyId, string selectedWasteType)
+        public async Task SaveWhatHaveYouDoneWaste(int journeyId, string whatHaveYouDoneWaste)
```

Comment:
As above, pass the view model in and put relevant checks in to ensure it's valid, throwing any exceptions as required.

Reply:
Done.